quickstarts/microsoft.sql/sql-logical-server-aad-only-auth/azuredeploy.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.1",
+    "parameters": {
+        "server": {
+            "type": "string",
+            "defaultValue": "[uniqueString('sql', resourceGroup().id)]",
+            "metadata": {
+                "description": "The name of the SQL logical server."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "aad_admin_name": {
+            "type": "String",
+            "metadata": {
+                "description": "The name of the Azure AD admin for the SQL server."
+            }
+        },
+        "aad_admin_objectid": {
+            "type": "String",
+            "metadata": {
+                "description": "The Object ID of the Azure AD admin."
+            }
+        },
+        "aad_admin_tenantid": {
+            "type": "String",
+            "metadata": {
+                "description": "The Tenant ID of the Azure Active Directory"
+            }
+        },
+        "aad_admin_type": {
+            "defaultValue": "User",
+            "allowedValues": [
+                "User",
+                "Group",
+                "Application"
+            ],
+            "type": "String"
+        },
+        "aad_only_auth": {
+            "defaultValue": true,
```

Comment:
what happens if this is set to false?  does the deployment fail or is the server created with no way to log into it?

Reply:
The deployment should not fail. This feature only disables SQL Authentication, but the Azure AD user that was added should still be able to log in.